src/components/Timelog/EffortBar.jsx

```diff
@@ -21,7 +21,7 @@ const EffortBar = ({activeTab, projectsSelected}) => {
     const totalTime = tangibleTime + intangibleTime;
        
     return (
-        <div className="m-2 row text-white text-center">
+        <div style = {{width: "96%", alignSelf: "center"}} className="m-auto row text-white text-center align-self-center">
```

Comment:
`<div className="m-auto row text-white text-center">`. This is enough for it to work.